runtime/j9vm/java11vmi.c

```diff
@@ -1455,8 +1455,10 @@ JVM_ConstantPoolGetClassRefIndexAt(JNIEnv *env, jobject arg1, jlong arg2, jint a
 
 jobjectArray JNICALL
 JVM_GetVmArguments(JNIEnv *env) {
-	assert(!"JVM_GetVmArguments unimplemented"); /* Jazz 108925: Revive J9JCL raw pConfig build */
-    return NULL;
+	/* call com.ibm.oti.vm.VM.getVMargs() */
+	jclass vmClass = (*env)->FindClass(env, "com/ibm/oti/vm/VM");
```

Comment:
what if `FindClass` fails?

Reply:
There's a macro to get the VM class from the vmconstantpool as we already know about this class.  Better to use that than JNI